test/abilities/contrary.test.ts

```diff
@@ -54,17 +54,14 @@ describe("Abilities - Contrary", () => {
     });
 
     it("should block negative effects", async () => {
-      game.override
-        .enemyPassiveAbility(Abilities.CLEAR_BODY)
-        .enemyMoveset([MoveId.HOWL, MoveId.HOWL, MoveId.HOWL, MoveId.HOWL])
-        .moveset([MoveId.SPLASH]);
+      game.override.enemyPassiveAbility(Abilities.CLEAR_BODY).moveset(MoveId.SWAGGER);
```

Comment:
```suggestion
      game.override
        .enemyPassiveAbility(Abilities.CLEAR_BODY)
        .moveset(MoveId.SWAGGER)
        .passiveAbility(Abilities.NO_GUARD);
```
(Swagger has less than 100% accuracy)